src/GHCMod.hs

```diff
@@ -277,20 +277,20 @@ File map docs:
 CLI options:
 * `--map-file "file1.hs=file2.hs"` can be used to tell
     ghc-mod that it should take source code for `file1.hs` from `file2.hs`.
-    `file1.hs` can be either full path, or path relative to project root.
+    `file1.hs` can be either full path, or path relative to current directory.
```

Comment:
Just checked and you're entirely right. I was mistaken. Will fix this to reflect reality. Sorry.